### PR TITLE
Fix some duplication of partials, and add fix versions for update-primary data loss issue

### PR DESCRIPTION
**Pull request by ncabatoff**

### CODE REVIEW

File: website/content/docs/release-notes/1.13.0.mdx [hunk 1]
```diff
@@ -75,7 +75,6 @@ Some of these enhancements and changes in this release include the following:
   that all TLS connections between the auth engine and the Kubernetes API were
   validated against the configured set of CA certificates.
 - **Kubernetes Secretes Engine on Vault UI:** Introducing Kubernetes secret
-  engine support on the UI
```
Comment (raskchanky) [inline website/content/docs/release-notes/1.13.0.mdx:78]: Was this intentional?

**Review by raskchanky — APPROVED** [review]
One small comment about a deletion that didn't make sense but looks good otherwise. The versions are correct, AFAICT.

File: website/content/docs/release-notes/1.13.0.mdx (hunk 1 above)
Comment (ncabatoff) [reply website/content/docs/release-notes/1.13.0.mdx:78]: No, ty!

**Review by schavis — APPROVED** [review]
LGTM

**Review by kubawi — COMMENTED** [review]
LGTM

**Pull request merged**